app/main/index.js

```diff
@@ -209,10 +209,10 @@ app.on('ready', () => {
 	});
 
 	ipcMain.on('toggle-app', () => {
-		if (mainWindow.isVisible()) {
-			mainWindow.hide();
-		} else {
+		if (!mainWindow.isVisible() || mainWindow.isMinimized() || !mainWindow.isFocused()) {
```

Comment:
@bemyak this doesn't work for me on Windows, can you keep the old method?

Reply:
Thank you!
I've tested it on Windows today. The problem was in `isFocused()` part of condition.
The reason is simple — when you click on tray on Windows the tray gets the focus for a second and app's window gets unfocused.
I remove the condition and now everything works as expected.